### ISSUE TITLE
Guid type as method parameter

### ISSUE THREAD

**Issue by sonicmouse** [jacqueskang/IpcServiceFramework#9]
Since Json converts GUID to string, when `DefaultValueConverter` tries to convert, there is no way to convert a string to GUID.

I added an explicit check to `DefaultValueConverter` for GUID types.

```
            if(destType == typeof(Guid))
            {
                if(Guid.TryParse(origValue.ToString(), out Guid guid))
                {
                    destValue = guid;
                    return true;
                }
                destValue = null;
                return false;
            }
```

**Comment by jacqueskang** [comment]
accidentally closed the issue before a new version is released

**Comment by jacqueskang** [comment]
Thanks @sonicmouse 
this changes is included in v1.0.3
